dm/dumpling/dumpling.go

```diff
@@ -196,7 +201,43 @@ func (m *Dumpling) Update(context.Context, *config.SubTaskConfig) error {
 // Status implements Unit.Status.
 func (m *Dumpling) Status(_ *binlog.SourceStatus) interface{} {
 	// NOTE: try to add some status, like dumped file count
-	return &pb.DumpStatus{}
+	mid := &export.Midparams{}
+	mid = m.realdumper.GetParameters()
```

Comment:
should add a test about calling `m.realdumper.GetParameters()` after `dumpling.Dump()`

Reply:
I agree and I will try to add this test